pkgs/desktops/xfce/core/thunar/default.nix

```diff
@@ -44,9 +44,9 @@ let unwrapped = mkXfceDerivation {
     xfconf
   ];
 
-  patches = [
-    ./thunarx_plugins_directory.patch
-  ];
+  configureFlags = [ "--with-custom-thunarx-dirs-enabled" ];
+
+  NIX_CFLAGS_COMPILE = "-I${libxfce4ui.dev}/include/xfce4";
```

Comment:
Not a change request: *I am quite curious in why this is now needed and I just filed https://gitlab.xfce.org/xfce/thunar/-/issues/972 (haven't filed an issue for `xfce4-terminal` yet but I think they are the same cause)* :thinking: